packages/synapse-react-client/src/lib/containers/LoadingScreen.tsx

```diff
@@ -69,38 +76,51 @@ export const BlockingLoader: React.FC<BlockingLoaderProps> = ({
           value={(currentProgress! / totalProgress!) * 100}
         />
       </div>
-      <Typography variant="headline3">{headlineText}</Typography>
-      <Typography variant="hintText">{hintText}</Typography>
+      <Typography variant="headline3" mb={2}>
+        {headlineText}
+      </Typography>
+      <Typography variant="hintText" mb={2}>
+        {hintText}
+      </Typography>
     </>
   )
   return (
-    <Modal
-      className="bootstrap-4-backport BlockingLoader"
-      backdrop={false}
-      animation={false}
-      show={show}
-      size="sm"
-      centered={true}
-      onHide={() => {}}
+    <Backdrop
+      open={show}
+      onClick={() => {}}
+      aria-hidden={false}
```

Comment:
Backdrop's default value for [aria-hidden is true](https://github.com/mui/material-ui/issues/33146). Update the value to ensure that components within the Backdrop are accessible when blocking the screen